docs/templates/README.md

```diff
@@ -0,0 +1,34 @@
+# Templates
+
+Find all provided documentation templates in this folder. Please note that the _italic text 
+and sentences_ should be removed. Feel free to add additional sections and subsections, however, make sure
+that at least the sections of the templates marked as "mandatory" are filled.
+
+## Decision Records
+
+[Link](decision-record.md) to template.
+
+Each decision record should be put in an appropriate folder that is following a naming pattern: 
+`YYYY-MM-DD-title-of-decision-record`. This should be located at the [decision record folder](../developer/decision-records) 
+and contain all relevant files, at least a filled-out template named `README.md` and any additional images.
+
+As of now, every merged decision record is in state `accepted`. Please make sure to add a comment to
+a decision record that replaces a previous one with adding a hint: `superseded by [...]`.
+
+## Extensions
+
+[Link](extension.md) to template.
+
+Every module located [in the extensions folder](../../extensions) has to provide documentation regarding its 
```

Comment:
nit: broken link. Pls ignore if we do not need this one here or we have some other ways to cover it from upstream.

Just adding the comment to highlight what is missing.

Reply:
removed edc specific section from file